### PR TITLE
GameDB: Serial update

### PR DESCRIPTION
**Pull request by atomic83GitHub**
- serials update along with a minor modification to siphon filter patches typo.

- Pac Man world 2 Timing fixes to avoid hangs at various levels. Tested by atomic83github.

- Missing VU clamping fixes for some EA sports titles. Tested by atomic83github.

Note: Pac man World 2 still hangs during tutorial.

### CODE REVIEW

File: bin/GameIndex.dbf [hunk 1]
```diff
@@ -44332,14 +44350,44 @@ Compat = 5
...
+Name   = Killzone Beta Trial Code
+Region = PAL-E
+vuClampMode = 0 // Resolves I Reg Clamping / performance impact and yellow graphics in certain areas.
+---------------------------------------------
+Serial = TCES-52033
+Name   = Syphon Filter: The Omega Strain Beta Trial Code
+Region = PAL-E
+EETimingHack = 1 // Fixes random hangs.
+[patches]
+	comment=If game broken, report to PCSX2 team.
```
Comment (lightningterror) [inline bin/GameIndex.dbf]: Kinda useless as a patch comment.

File: bin/GameIndex.dbf [hunk 2]
```diff
@@ -2863,8 +2865,10 @@ Region = NTSC-K
 Serial = SCKA-20032
 Name   = Syphon Filter - The Omega Virus
 Region = NTSC-K
-EETimingHack = 1 // Random hangs.
+EETimingHack = 1 // Fixes Random hangs.
```
Comment (lightningterror) [inline bin/GameIndex.dbf]: If you like you can change Random not to use capital letter, your choice since it's a very small nitpick.

**Comment by prafullpcsx2** [comment]
Ee cycle rate - 1 doesn't fix pac man world 2?

**Comment by atomic83GitHub** [comment]
Yes it does but EE timing hack have the exact same effect as ee cyclerate but without occasional hicups due to reduced cyclerate.

File: bin/GameIndex.dbf (hunk 2 above)
Comment (atomic83GitHub) [reply bin/GameIndex.dbf]: Done

**Pull request merged**